Mathlib/Algebra/Category/ModuleCat/Monoidal/Closed.lean

```diff
@@ -106,4 +107,5 @@ theorem monoidalClosed_pre_app {M N : ModuleCat.{u} R} (P : ModuleCat.{u} R) (f
 set_option linter.uppercaseLean3 false in
 #align Module.monoidal_closed_pre_app ModuleCat.monoidalClosed_pre_app
 
+attribute [nolint simpNF] ModuleCat.monoidalClosed_curry
```

Comment:
Here we have a problem with `ModuleCat.carrier R CommRing.toRing (op M).unop` vs just `M`